itests/org.openhab.binding.nest.tests/itest.bndrun

```diff
@@ -65,6 +65,7 @@ Fragment-Host: org.openhab.binding.nest
 	org.eclipse.jetty.websocket.client;version='[9.4.20,9.4.21)',\
 	org.eclipse.jetty.websocket.common;version='[9.4.20,9.4.21)',\
 	org.eclipse.jetty.xml;version='[9.4.20,9.4.21)',\
-	ch.qos.logback.classic;version='[1.2.3,1.2.4)',\
```

Comment:
logback is still used in the modbus tests, so I don't know if you forgot to switch it or had another reason to leave it...